ietf/templates/doc/badge/doc-badge-rfc.html

```diff
@@ -0,0 +1,13 @@
+{% load origin %}
+{% load static %}
+{% load ietf_filters %}
+{% load person_filters %}
+{% origin %}
+
+<span class="text-success" style="border: dashed 3px green">RFC
+    {% if not document_html %}
+    - {{ doc.std_level }}
+    {% else %}
+        <span class="badge rounded-pill badge-{% if not snapshot %}{{ doc|std_level_to_label_format }}{% else %}draft{% endif %}">{{ doc.std_level }}</span>
```

Comment:
Seems wrong to fall back to a class of "badge-draft". Probably needs a new class to fall back to.

Reply:
Have chosen 'badge-generic' as the fallback. Naming improvements welcome :)